core/src/monetaryvalue.test.ts

```diff
@@ -38,9 +38,7 @@ test("MonetaryValue", async (t) => {
 
   await t.test("add", async (t) => {
     await t.test("with MonetaryValue", () => {
-      const mv = new MonetaryValue()
-        .add(new MonetaryValue("blue", 2))
-        .add(new MonetaryValue("red", 2));
+      const mv = new MonetaryValue().add(new MonetaryValue("blue", 2)).add(new MonetaryValue("red", 2));
```

Comment:
I found that prettier would try to automatically unwrap the lines 🤔 
and [stackoverflow](https://stackoverflow.com/questions/69499723/how-to-configure-prettier-to-stop-unwrapping-my-intended-wrapped-lines) states that there are not much I can do to prevent that except for ugly comments @@?

Reply:
Haha yeah that's why I added those empty comments in the tests for MonetaryValue